test/go.test.ts

```diff
@@ -380,14 +380,18 @@ It returns the number of bytes written and any write error encountered.
 				{ line: 11, severity: 'warning', msg: 'error return value not checked (undeclared name: prin) (errcheck)' },
 				{ line: 11, severity: 'warning', msg: 'unused variable or constant undeclared name: prin (varcheck)' },
 			];
-			return check(vscode.Uri.file(path.join(fixturePath, 'errorsTest', 'errors.go')), config).then(diagnostics => {
+			let errorsTestPath = path.join(fixtureSourcePath, 'errorsTest', 'errors.go');
```

Comment:
The fixtures from `fixtureSourcePath` get copied to `fixturePath` in lines https://github.com/Microsoft/vscode-go/blob/0.6.77/test/go.test.ts#L46-L71. So why this change?

Reply:
It was failing locally but it was probably result of not having the linters installed instead of the path. My bad

Reply:
`fixtureSourcePath` is still in use here and in line 856. Can you revert that back to `fixturePath`?

Reply:
Yeah, got it backwards. You may squash all of these commits upon merge